components/IssueCredentialOptions.yml

```diff
@@ -33,6 +33,9 @@ components:
             type:
               type: string
               description: The type of credential status to issue the credential with
+        credentialId:
+          type: string
+          description: An identifier that can be used to identify the credential. credentialId SHOULD NOT be set if the id property is set. If credential.id is set, that value will be duplicated for credentialId. If neither credentialId nor id is set, there will be no way to uniquely identify a credential or deal with duplicate errors. This property should not be used as a replacement for credential.id, rather as a means of identifying a credential without the id property set.
```

Comment:
```suggestion
          description: An identifier that can be used to identify the credential. credentialId SHOULD NOT be set if the id property is set. If credential.id is set, that value will be duplicated for credentialId. If neither credentialId nor id is set, there will be no way to uniquely identify a credential or deal with duplicate errors. This property should not be used as a replacement for credential.id, rather as a means of identifying a credential without the id property set. The issuer SHOULD NOT auto-generate a credentialId because doing so could create a partitioning error if the result is never received by the client.
```